src/script/lua_api/l_areastore.cpp

```diff
@@ -129,45 +129,46 @@ int LuaAreaStore::l_get_areas_for_pos(lua_State *L)
 
 	v3s16 pos = check_v3s16(L, 2);
 
-	bool include_borders = true;
+	bool include_corners = true;
 	bool include_data = false;
-	get_data_and_border_flags(L, 3, &include_borders, &include_data);
+	get_data_and_corner_flags(L, 3, &include_corners, &include_data);
 
 	std::vector<Area *> res;
 
 	ast->getAreasForPos(&res, pos);
-	push_areas(L, res, include_borders, include_data);
+	push_areas(L, res, include_corners, include_data);
 
 	return 1;
 }
 
-// get_areas_in_area(edge1, edge2, accept_overlap, include_borders, include_data)
+// get_areas_in_area(corner1, corner2, accept_overlap, include_corners, include_data)
 int LuaAreaStore::l_get_areas_in_area(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
 
 	LuaAreaStore *o = checkobject(L, 1);
 	AreaStore *ast = o->as;
 
-	v3s16 minedge = check_v3s16(L, 2);
-	v3s16 maxedge = check_v3s16(L, 3);
+	v3s16 minp = check_v3s16(L, 2);
+	v3s16 maxp = check_v3s16(L, 3);
+	sortBoxVerticies(minp, maxp);
```

Comment:
Sorting happens in `Area::Area()`, although this function does not make use of this class to sanitize user input.